src/main/java/org/example/jvspringbootfirstbook/service/order/OrderServiceImpl.java

```diff
@@ -29,8 +28,9 @@
 @Service
 @RequiredArgsConstructor
 public class OrderServiceImpl implements OrderService {
+    public static final String MISSING_ORDER_MESSAGE = "Order with your id not found";
+    public static final String MISSING_ORDER_ITEM_MESSAGE = "Order item with your id not found";
```

Comment:
You should refactor these constants so that at the end in exceptions you can add information about id.

Reply:
Like this for example:
"Order not found for id: "